scripts/servers.py

```diff
@@ -380,20 +380,31 @@ def create_managed_web_browser(
     Returns:
         context manager|None. The context manager to a web browser window, or
         None if the current operating system does not support web browsers.
+
+    Raises:
+        Exception. It can raise various exceptions such as FileNotFound
+            or the process failure exceptions.
     """
     url = 'http://localhost:%s/' % port
     human_readable_name = 'Web Browser'
-    if common.is_linux_os():
-        if any(re.match('.*VBOX.*', d) for d in os.listdir('/dev/disk/by-id/')):
-            return None
-        else:
+    try:
+        if common.is_linux_os():
+            if any(re.match('.*VBOX.*', d) for d in
```

Comment:
I thought you were planning to modify this to get rid of this special-casing logic?

Reply:
I wanted to confirm with you if it is okay to remove the existing code. I wasn't sure.

Reply:
It makes sense. I removed the code.